homeassistant/components/kaleidescape/media_player.py

```diff
@@ -0,0 +1,217 @@
+    kaleidescape_const.VIDEO_COLOR,
+]
+
+KALEIDESCAPE_PLAYING_STATES = [
+    kaleidescape_const.PLAY_STATUS_PLAYING,
+    kaleidescape_const.PLAY_STATUS_FORWARD,
+    kaleidescape_const.PLAY_STATUS_REVERSE,
+]
+
+KALEIDESCAPE_PAUSED_STATES = [kaleidescape_const.PLAY_STATUS_PAUSED]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the platform from a config entry."""
+    entities = [KaleidescapeMediaPlayer(hass.data[KALEIDESCAPE_DOMAIN][entry.entry_id])]
+    async_add_entities(entities, True)
+
+
+class KaleidescapeMediaPlayer(MediaPlayerEntity):
+    """Representation of a Kaleidescape device."""
+
+    def __init__(self, device: KaleidescapeDevice) -> None:
+        """Initialize media player."""
+        self._device = device
+
+        self._attr_should_poll = False
+        self._attr_unique_id = device.serial_number
+        self._attr_name = f"{KALEIDESCAPE_NAME} {device.system.friendly_name}"
+        self._attr_supported_features = SUPPORTED_FEATURES
+
+    async def async_added_to_hass(self) -> None:
+        """Register update listener."""
+
+        @callback
+        def _update(event: str) -> None:
+            """Handle device state changes."""
+            self.async_write_ha_state()
+
+        self.async_on_remove(self._device.dispatcher.connect(_update).disconnect)
+
+    async def async_turn_on(self) -> None:
+        """Send leave standby command."""
+        await self._device.leave_standby()
+
+    async def async_turn_off(self) -> None:
+        """Send enter standby command."""
+        await self._device.enter_standby()
+
+    async def async_media_pause(self) -> None:
+        """Send pause command."""
+        await self._device.pause()
+
+    async def async_media_play(self) -> None:
+        """Send play command."""
+        await self._device.play()
+
+    async def async_media_stop(self) -> None:
+        """Send stop command."""
+        await self._device.stop()
+
+    async def async_media_next_track(self):
+        """Send track next command."""
+        await self._device.next()
+
+    async def async_media_previous_track(self):
+        """Send track previous command."""
+        await self._device.previous()
+
+    @property
+    def available(self) -> bool:
+        """Return if device is available."""
+        return self._device.is_connected
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return device specific attributes."""
+        return DeviceInfo(
+            identifiers={(KALEIDESCAPE_DOMAIN, self._device.serial_number)},
+            name=self.name,
+            model=self._device.system.type,
+            manufacturer=KALEIDESCAPE_NAME,
+            sw_version=f"{self._device.system.kos_version}",
+            suggested_area="Theater",
+            configuration_url=f"http://{self._device.host}",
+        )
```

Comment:
You can set this as `self._attr_device_info`

Reply:
Moved.